manifests/init.pp

```diff
@@ -103,7 +105,7 @@
   String $repo_version = $::katello::params::repo_version,
   Boolean $repo_gpgcheck = $::katello::params::repo_gpgcheck,
   Optional[String] $repo_gpgkey = $::katello::params::repo_gpgkey,
-
+  Integer[0] $rest_client_timeout = $::katello::params::rest_client_timeout,
```

Comment:
The settings were grouped. This block is for candlepin. Could you move this to the block with post_sync_token?